### DISCUSSION TITLE
Can't find dataset option during learning tictoc 18

### DISCUSSION THREAD

**Discussion by Sable911** [omnetpp/omnetpp#1064] [Q&A] [answered]
- OMNeT++ version: [6.0.1]

Can someone help me? I'm learning tictoc 18, following Tutorials of omnet++, but I can't find the dataset option in an anf file. There's only a "chart" option instead of "dataset and charts". ↓

![image](https://github.com/omnetpp/omnetpp/assets/133728058/cfc5456e-9f54-4688-8560-7f1d48f448b2)

And here's what it looks like in the Tutorials： ↓

![image](https://github.com/omnetpp/omnetpp/assets/133728058/3952b6ef-3d8d-4dba-a26b-d82aab8dee6f)
![image](https://github.com/omnetpp/omnetpp/assets/133728058/eb730a11-c8bd-4d51-bde8-6868f42f12c0)

**Comment by torokati44** [comment, marked as answer]
Thank you for reporting this! The result analysis tool of the OMNeT++ IDE was entirely overhauled in the 6.0 version, but the TikTok tutorial hasn't been updated yet, this is why you can't find what's in there. We will update the tutorial soon to match the new version of the tool.

**Reply by Sable911** [reply]
Thank you !!
